core/domain/state_domain.py

```diff
@@ -2433,6 +2740,32 @@ def to_dict(self):
             'rule_type': self.rule_type
         }
 
+    @classmethod
+    def to_written_translatable_text_proto(cls, translation):
+        """Creates a WrittenTranslatableText proto object.
+
+        Args:
+            translation: str. The translated content.
+
+        Returns:
+            WrittenTranslatableText. The proto object.
+        """
+        return languages_pb2.WrittenTranslatableTextDto(translation=translation)
+
+    @classmethod
+    def to_written_translatable_set_proto(cls, translation):
+        """Creates a SetOfWrittenTranslatableText proto object.
+
+        Args:
+            translation: list(str). The translated content.
+
+        Returns:
+            WrittenTranslatableSetDto. The proto object.
+        """
+        return languages_pb2.SetOfWrittenTranslatableTextDto(
+            translations=translation
```

Comment:
Should the arg be `translations` instead?

Reply:
Done